### DISCUSSION TITLE
Daily 23/9

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [rpgrca/proyecto-ispc-fullstack#31] [General]
Ayer terminé con la lectura de los contenidos de SGC habilitados hasta el momento. Repasé un poco de CSS. Participé de las clases sincrónicas de SGC y de Full Stack. Intercambiamos sobre las tareas para el próximo Sprint. Y finalmente continué con el repaso de Inglés.
Hoy participaré de la clase sincrónica de Inglés. Estare Leyendo los contenidos de CSS y practicando los mismo en VSC. Vere el video de la clase sincrónica de Full Stack del 19/09 donde se explica sobre el Proyecto.
No veo dificultades para el día de hoy.

**Comment by rpgrca** [comment]
Ayer rendí el examen de inglés, estuve en la clase de SGC y también me colé en la clase de full-stack que siguió. Armé más historias de usuario y otras issues, renombré las anteriores para que usen el sistema que quieren ellos (USxx para historias de usuario, TKxx para las issues). Hoy quiero ver si puedo terminar el documento de especificaciones que necesita dos secciones más.

**Comment by AhumadaCeleste** [comment]
Ayer realice el examen de ingles, y avance un poco con HTML. Hoy me asigne Issues en Git referentes al proyecto y maquete ya algunas. Me falta subirlas. Ahora veré la clase de SGC y la de FullStack de esta semana. Mañana Esperó terminar con HTML y 
comenzar con css.

**Comment by IngridAlcocer05** [comment]
Ayer finalice la lectura de HTML y repase ingles. Hoy empecé con css, continúe con python y instale Wordpress.

**Comment by claubott** [comment]
Ayer asistí a los encuentros sincrónicos. Hoy estoy empezando con la guia POO.
